db/migrate/20211207103201_add_user_id_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToArticles < ActiveRecord::Migration[6.1]
+  def change
+    add_column :articles, :user_id, :int
```

Comment:
i think it's better to use `bigint` rather than `int` to the user_id column, also we can add the FK for that column, senpaii <3